src/components/AttentionBox/__stories__/attentionBox.stories.mdx

```diff
@@ -0,0 +1,162 @@
+          </div>,
+        description: "Use when you are speaking directly to a piece of content and the notification can be positioned close to the content."
+      },
+      negative: {
+        component:
+          <div>
+            <span className="monday-style-attention-box_title">Inbox</span>
+            <span className="monday-style-attention-box_text" >Catch up on updates from all your boards.</span>
+            <AttentionBox text={<span className="monday-style-attention-box_info">7 days left on your free trial <Link>Learn More</Link></span>} compact componentClassName="monday-style-attention-box_att-box" />
+          </div>,
+        description: <>Don’t use when a message applies at a system level, does not apply to a specific piece of content, or needs to alert all users. On those cases, use <Link href="?path=/docs/components-alertbanner--overview" withoutSpacing>Alert banner.</Link></>
+      }
+    }
+  ]}
+/>
+
+## Use cases and examples
+### Natural attention box
+Provides additional information about an action or section.
+<Canvas>
+  <Story name="Natural attention box">
+    <div className="monday-style-attention-box_wrapper">
+      <span className="monday-style-attention-box_title">Cross-Account Copier</span>
+      <span className="monday-style-attention-box_text" >Copy boards and dashboards to another account</span>
+      <AttentionBox 
+        text={<span className="monday-style-attention-box_inline-wrapper"><Icon icon={Info} iconSize="18" />First, move the content you want to copy into folder. Only main boards and dashboards can be copied.<Link href="#" size={Link.sizes.SMALL} withoutSpacing>Learn more.</Link></span>}
+        type={ATTENTION_BOX_TYPES.DARK}
+      />
+    </div>
+  </Story>
+</Canvas>
+
+### Attention box inside a dialog/ combo box
+Provides cotextual and related information.
+<Canvas>
+  <Story name="Attention box inside a dialog/ combo box">
+    <DialogContentContainer className="monday-style-attention-box_search-bar">
+      <Search placeholder="Search by name, role, team, or email" />
+      <div className="monday-style-attention-box_lable">Suggested people</div>
+      <div className="monday-style-attention-box_search">
+        <div className="monday-style-attention-box_inline-container">
+          <Avatar size={Avatar.sizes.MEDIUM} src={person} type={Avatar.types.IMG} />
+          <span className="monday-style-attention-box_name">May Kishon<span>(UX/UI Product Designer)</span></span>
+        </div>
+        <div className="monday-style-attention-box_inline-container">
+          <Icon iconSize="24" icon={Invite} className="monday-style-attention-box_icon" />
+          <span className="monday-style-attention-box_name">Invite new board member by email</span>
+        </div>
+         <AttentionBox 
+        text={<span>Hold ⌘ to select more than one person or team</span>}
```

Comment:
please pass string text prop to the attention text component and not an element prop